clients/lsp-clangd.el

```diff
@@ -177,14 +178,24 @@ This must be set only once after loading the clang client.")
   :risky t
   :type '(repeat string))
 
+(defun lsp-clients--clangd-darwin-executable-find (executable &rest args)
```

Comment:
This is not only applied for darwin but actually anywhere. So we should name this as something more generally.
```suggestion
(defun lsp-clients--clangd-execute (executable &rest args)
```